Apps.DeepL/Actions/GetUsageAction.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Apps.DeepL.Requests;
+using Apps.DeepL.Responses;
+using Blackbird.Applications.Sdk.Common;
+using Blackbird.Applications.Sdk.Common.Actions;
+using Blackbird.Applications.Sdk.Common.Invocation;
+using Blackbird.Applications.SDK.Extensions.FileManagement.Interfaces;
+using DeepL;
+
+namespace Apps.DeepL.Actions;
+
+[ActionList]
+public class GetUsageActionActions : DeepLInvocable
```

Comment:
Action class is meant to contain actions for a specific entity and not only for one method. So the naming should be like "UsageActions"